### ISSUE TITLE
[bug] Issue when running latest 0.1.2 version

### ISSUE THREAD

**Issue by gosseti** [tmm/wagmi#60]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package Version

0.1.2

### Current Behavior

When installing the latest version (in a nextjs 11 app), I get the error:

```
error - Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './connectors/injected' is not defined by "exports" in /Users/gosset/Dev/frontend/node_modules/wagmi/package.json
```

### Steps To Reproduce

Install the latest version of the package inside of a nextjs 11 app and then run it.

**Comment by woodylewis** [comment]
Hi, I love the look of this project so far. I added ethers and wagmi to my well-travelled React TS app, got the errors listed below when running yarn start. Anything obvious I may have overlooked? Thanks.

ERROR in [at-loader] ./node_modules/wagmi-private/dist/declarations/src/types/index.d.ts:4:11 
    TS2304: Cannot find name 'AddEthereumChainParameter'.

ERROR in [at-loader] ./node_modules/wagmi-private/dist/declarations/src/types/index.d.ts:5:22 
    TS2304: Cannot find name 'AddEthereumChainParameter'.

ERROR in [at-loader] ./node_modules/wagmi-private/dist/declarations/src/types/index.d.ts:6:14 
    TS2304: Cannot find name 'AddEthereumChainParameter'.

ERROR in [at-loader] ./node_modules/wagmi-private/dist/declarations/src/connectors/injected.d.ts:3:65 
    TS2339: Property 'ethereum' does not exist on type 'Window'.

ERROR in [at-loader] ./node_modules/wagmi-private/dist/declarations/src/connectors/injected.d.ts:22:36 
    TS2304: Cannot find name 'RequestArguments'.

ERROR in [at-loader] ./node_modules/wagmi-private/dist/declarations/src/connectors/injected.d.ts:33:32 
    TS2304: Cannot find name 'RequestArguments'.

**Comment by tmm** [comment]
@gosseti haven't been able to reproduce. what version of wagmi were you using before upgrading?

have you tried removing wagmi completely and reinstalling?

**Comment by tmm** [comment]
released `wagmi@0.1.4`

if this is still an issue, feel free to re-open.
